tools/serverpod_cli/lib/src/config/config.dart

```diff
@@ -231,6 +238,19 @@ class GeneratorConfig {
     );
   }
 
+  static PackageType getPackageType(Map<dynamic, dynamic> generatorConfig) {
+    var typeStr = generatorConfig['type'];
+    late PackageType type;
+    if (typeStr == 'module') {
+      type = PackageType.module;
+    } else if (typeStr == 'internal') {
+      type = PackageType.internal;
+    } else {
+      type = PackageType.server;
+    }
```

Comment:
We should add some kind of throw here for the users but we won't do it now. Right now they won't be alerted if they have misspelled the type for a module and it will function as server.